router/routes.go

```diff
@@ -2,30 +2,33 @@ package router
 
 import (
 	docs "github.com/forjadev/gun-organization/docs"
-	"github.com/forjadev/gun-organization/handler"
 	"github.com/gin-gonic/gin"
 	swaggerfiles "github.com/swaggo/files"
 	ginSwagger "github.com/swaggo/gin-swagger"
+	"os"
 )
 
-// initializeRoutes sets up the routes for the application
-func initializeRoutes(r *gin.Engine) {
-	// Set the base path for all routes
+func InitializeRoutes() {
+	r := gin.Default()
 	basePath := "/api/v1"
 
-	// Set the base path for Swagger documentation
 	docs.SwaggerInfo.BasePath = basePath
 
-	// Create a new router group for version 1 of the API
-	v1 := r.Group(basePath)
+	r.GET("/swagger/*any", ginSwagger.WrapHandler(swaggerfiles.Handler))
 
-	// Define the routes for version 1 of the API
+	apiGroup := r.Group(basePath)
 	{
-		// Define a GET route for the /ping endpoint
-		v1.GET("/ping", handler.PingServerHandler)
-		v1.POST("/webhook", handler.GitHubWebhookHandler)
+		bindActuatorsRoutes(apiGroup)
+		bindWebhookRoutes(apiGroup)
 	}
 
-	// Initialize Swagger documentation
-	r.GET("/swagger/*any", ginSwagger.WrapHandler(swaggerfiles.Handler))
+	port := os.Getenv("PORT")
+	if port == "" {
+		port = "8080"
+	}
+
+	err := r.Run("0.0.0.0:" + port)
+	if err != nil {
```

Comment:
That's a nice addition. We didn't check for this error before. I think that this can happen when the port is already binded, for example.